templates/company-board/README.md

```diff
@@ -75,14 +74,13 @@ The network details will be automatically selected by the `arapp.json`'s environ
 | App                 | Permission            | Grantee             | Manager       |
 |---------------------|-----------------------|---------------------|---------------|
 | Agent               | RUN_SCRIPT            | Board Voting        | Share Voting  |
-| Agent               | RUN_SCRIPT            | Share Voting        | Share Voting  |
 | Agent               | EXECUTE               | Board Voting        | Share Voting  |
-| Agent               | EXECUTE               | Share Voting        | Share Voting  |
 
 ### Additional permissions if the Payroll app is installed
 
 | App                 | Permission                 | Grantee             | Manager       |
 |---------------------|----------------------------|---------------------|---------------|
+| Finance             | CREATE_PAYMENTS            | Payroll             | Board Voting  |
```

Comment:
Permission manager for `CREATE_PAYMENTS` should be `Share Voting`.
* see line 64
* code looks good - perms. mgr is transferred to `Share Voting`

Reply:
Good catch! Opened #153 for it